Assets/GothicVR/Scripts/Player/Camera/CameraFade.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace GVR
+{
+    public class CameraFade : MonoBehaviour
```

Comment:
Let's rework this a little, please.
As this seems to be a Singleton, we have two ways of handling this:
1. It's a static class, if we have no Unity calls at all (e.g. no Start or wired properties at Editor time)
2. Use a SingletonBehaviour<*> --> As you wire an image at editor time, I think we will use this.

SingletonBehaviour:
Look at e.g. FeatureFlags. They're also a SingletonBehavriour. In the end it's just a convenient wrapper to ensure:
1. We have an instance created only once in our scene
2. We have a fast lookup: e.g. FeatureFlags.I.*

Reply:
used SingletonBehaviour<*>